start_eudico.sh

```diff
@@ -28,28 +28,23 @@ chmod 600 /root/.eudico/keystore/*
 MUL=$(echo $HOSTNAME | sed 's/[^0-9]//g')
 
 eudico delegated daemon --genesis=gen.gen &
-eudico wait-api
+eudico wait-api --timeout 90s
 
+# Append own listening address to file
+eudico net listen | head -n 1 >> /config/peerID.txt
 
-if [ -s /config/peerID.txt ]; then
-        # Connect to all the nodes
+# Connect to other nodes
+sleep $((10*MUL))
```

Comment:
I don't think we actually need this sleep (we're merely parallel reading and issuing connects -- this would never result in overloading libp2p buffers for realistic node counts) but maybe it was here for a reason. In any case, it shouldn't hurt.